autodoc raises NotImplementedError for non-solidity domain

Issue: cannot document Solidity and Python sources in the same project.

---

With `sphinxcontrib.soliditydomain` added to the extensions, auto-documenting Solidity sources works, but auto-documenting a Python class fails with:

```
reading sources... [ 80%] api/xbrpy                                                                                                                                                                                                                                                
Exception occurred:
  File "/home/oberstet/cpy3_1/lib/python3.7/site-packages/sphinx/ext/autodoc/__init__.py", line 296, in can_document_member
    raise NotImplementedError('must be implemented in subclasses')
NotImplementedError: must be implemented in subclasses
The full traceback has been saved in /tmp/sphinx-err-30qmik_i.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:156: recipe for target 'docs' failed
make: *** [docs] Error 2
```

I used the following trivial class to track down this issue:

```python
class Test(object):
    def __init__(self, foo):
        """

        :param foo:
        """
        self.foo = foo

    def add2(self, x, y):
        """

        :param x:
        :param y:
        :return:
        """
        return x + y
```

and the following directive in a RST file:

```rst
.. autoclass:: xbr.Test
    :members: add2
```

---

Could it be that `sphinxcontrib.soliditydomain` is overwriting something that it shouldn't?

--- a/sphinxcontrib/soliditydomain/documenters.py
+++ b/sphinxcontrib/soliditydomain/documenters.py
@@ -15,6 +15,14 @@ class SolidityObjectDocumenter(Documenter):
         'exclude-members': members_set_option,
     }
 
+    @classmethod
+    def can_document_member(cls, member, membername, isattr, parent):
+        # type: (Any, unicode, bool, Any) -> bool
+        """Called to see if a member of a Python object can be documented by this documenter.
+
+        Will always answer no because this Documenter is built for Solidity."""
+        return False
+
     def get_sourcename(self):
         return '{}:docstring of {} {}'.format(
             self.object.file,

